tests/lib/browser.spec.js

```diff
@@ -0,0 +1,110 @@
+const {
+  mockPuppeteer,
+  mockPuppeteerLaunch,
```

Comment:
Declared variable but never used.

Reply:
I will add eslint to avoid those cases...

Reply:
Done.